### ISSUE TITLE
Is tf.train.SyncReplicasOptimizer able to do cross GPU cross worker batchnorm

### ISSUE THREAD

**Issue by MrWanter** [tensorflow/models#4654]

**Comment by ymodak** [comment]
Closing this issue due to lack of information to work with. Please open a new issue and provide all the information asked by the template. Thanks!
